src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { PrevArrow } from '../icons/buttons-icons/prev-arrow';
+import { NextArrow } from '../icons/buttons-icons/next-arrow';
+import { theme } from '../layout/theme';
+import { PaginationDropdown } from '../typography/typography';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+
+interface PaginationProps {
+  onPreviousClick: () => any;
+  onNextClick: () => any;
+  onDropdownSelect: (selectedPage: number) => any;
+  amountOfPages: number;
+  currentPage: number;
+}
+
+const PaginationContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Dropdown = styled(PaginationDropdown)`
+  background: none;
+  border: none;
+  color: ${theme.palette.text.link.default};
+  cursor: pointer;
+`;
+
+const StyledButton = styled.button<{ inactive: boolean }>`
```

Comment:
For buttons that are not active the disabled flag should be used: https://www.w3schools.com/tags/att_disabled.asp

- The property should also be rendered in HTML, which makes it accessible for screen readers and other tools that detect "inactive" buttons. Best to do a double-check if this property is really set. 
- With this property, you can also use the native `:disabled` CSS selector instead of the prop.
- No extra checks in your `onClick` handler is necessary anymore, as the browser won't let you click disabled buttons in the first place.

Reply:
Thanks, good hint 👍 The component has also become much clearer. I did a double-check and the flag works fine:
<img width="457" alt="Bildschirmfoto 2021-11-23 um 08 33 20" src="https://user-images.githubusercontent.com/57712895/142986141-76ffdc57-93f6-4c02-a8fd-5c9f4dfbd3e0.png">